### ISSUE TITLE
Add New Book Feature

### ISSUE THREAD

**Issue by paulsaenzsucre** [paulsaenzsucre/Bookstore#19]
In the `AddBookForm` component, add functionality for "Add new" button click:

- [x] Import Redux `useDispatch`.
- [x] Set _title_ and _author_ in the local React state (NOTE: categories and comments are NOT part of this step).
- [x] Take the inputs from the state then generate a unique id and add them to an object.
- [x] Once your new book object is ready to be submitted to Redux store, dispatch the corresponding action.
- [x] Keep state immutable in the reducer.
